src/topology/constructions.lean

```diff
@@ -641,6 +641,10 @@ continuous_induced_dom
 lemma continuous_subtype_coe : continuous (coe : subtype p → α) :=
 continuous_subtype_val
 
+lemma continuous.subtype_coe {f : β → subtype p} (hf : continuous f) :
```

Comment:
```suggestion
@[continuity] lemma continuous.subtype_coe {f : β → subtype p} (hf : continuous f) :
```

Reply:
Adding this attribute breaks some `continuity` proofs, so I removed it again.

Reply:
Maybe worth adding a comment to that effect, since I could imagine someone trying to add the attribute again.

Reply:
Is that true? Looking at this file, no other declaration like this has continuity attribute. E.g. `continuous_snd` has one, but `continuous.snd` doesn't. So if you just mimic what is already in the file, it should go fine.
(Also, in such a comment I'd like to see the "why" question answered, which I cannot answer myself.)

Reply:
Indeed they don't have such attributes. That surprises me. I made the `measurability` tactic by copying `continuity` and all lemmas like `measurable.subtype_coe` are now tagged, which works fine. I guess that we went for a different way of tagging things (without knowing that it was different) and I don't have a clue about why we should use one or the other.

Let's have that lemma without attribute then.